packages/material-ui/src/Slider/Slider.js

```diff
@@ -720,7 +721,17 @@ Slider.propTypes = {
   /**
    * The label of the slider.
    */
-  'aria-label': PropTypes.string,
+  'aria-label': chainPropTypes(PropTypes.string, props => {
+    const range = Array.isArray(props.value || props.defaultValue);
+
+    if (range && props['aria-label']) {
+      return new Error(
+        'Material-UI: you need to use the `getAriaLabel` prop instead of `aria-label` when using a range input.',
```

Comment:
should this say `when using a range slider` instead?

Reply:
Yeah, this sounds better, for both occurrences